clickhouse_search/models.py

```diff
@@ -707,42 +722,104 @@ def _save_table(
             raw=raw, cls=cls, force_insert=True, force_update=force_update, using=using, update_fields=update_fields,
         )
 
+    class Meta:
+        abstract = True
 
-class TranscriptsSnvIndel(models.ClickhouseModel):
-    key = OneToOneField('AnnotationsSnvIndel', db_column='key', primary_key=True, on_delete=CASCADE)
-    transcripts = NestedField([
-        ('alphamissense', NamedTupleField([
-            ('pathogenicity', models.DecimalField(null=True, blank=True, max_digits=9, decimal_places=5)),
-        ])),
+
+class BaseEntriesSnvIndel(BaseEntries):
+    CALL_FIELDS = [
+        ('sampleId', models.StringField()),
+        ('gt', models.Enum8Field(null=True, blank=True, choices=[(0, 'REF'), (1, 'HET'), (2, 'HOM')])),
+        ('gq', models.UInt8Field(null=True, blank=True)),
+        ('ab', models.DecimalField(max_digits=9, decimal_places=5, null=True, blank=True)),
+        ('dp', models.UInt16Field(null=True, blank=True)),
+    ]
+    is_gnomad_gt_5_percent = models.BoolField()
+    calls = models.ArrayField(NamedTupleField(CALL_FIELDS))
+
+    class Meta:
+        abstract = True
+        engine = CollapsingMergeTree(
+            'sign',
+            order_by=('project_guid', 'family_guid', 'is_gnomad_gt_5_percent', 'key'),
+            partition_by='project_guid',
+            deduplicate_merge_projection_mode='rebuild',
+            index_granularity=8192,
+        )
+        projection = Projection('xpos_projection', order_by='xpos, is_gnomad_gt_5_percent')
+
+class EntriesGRCh37SnvIndel(BaseEntriesSnvIndel):
+
+    objects = EntriesManager()
```

Comment:
you can put this on the base class instead of putting it explicitly on both inherited classes